cli53: add page (#11361)
tldr-pages/tldr: pages/common/cli53.md

# cli53

> Command line tool for Amazon Route 53.
> More information: <https://github.com/barnybug/cli53>.

- List domains:

`cli53 list`

- Create a domain:

`cli53 create {{mydomain.com}} --comment "{{comment}}"`

- Export a bind zone file to stdout:

`cli53 export {{mydomain.com}}`

- Create a `www` subdomain pointing to a relative record in the same zone:

`cli53 {{rc|rrcreate}} {{mydomain.com}} {{'www 300 CNAME lb'}}`

> - Create a `www` subdomain pointing a external address (must ends with a dot):

Reviewer: ```suggestion
- Create a `www` subdomain pointing an external address (must end with a dot):
```

> - Create a `www` subdomain pointing an external address (must end with a dot):

Reviewer 2: ```suggestion
- Create a `www` subdomain pointing to an external address (must end with a dot):
```

`cli53 {{rc|rrcreate}} {{mydomain.com}} {{'www 300 CNAME lb.externalhost.com.'}}`

> - Create a `www` subdomain pointing to an IP:

Reviewer: ```suggestion
- Create a `www` subdomain pointing to an IP address:
```
